### PR TITLE
Module 4

### PR DESCRIPTION
**Pull request by kateblankenship1**
Pull request.

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        <li><a href="dogs.html">Dogs /</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </div>
+
+<div class="blogheader">
+  <h2>Adoptadog Blog</h2>
+</div>
+
+<article class="blogarticles">
```
Comment (jimmynono) [inline blog.html]: when using multi word css classes it's best to use kabob casing, where everything is lowercase and words are seperated by a dash.

```
<article class="super-exciting-fun-times">

File: blog.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      <h3 class="blogtitle">Traveling With Your Dog</h3>
+        <p> Doggo ipsum dat tungg tho doggorino maximum borkdrive, yapper. Thicc what a nice floof very hand that feed shibe puggo
+          snoot borking doggo, puggorino ur givin me a spook noodle horse. Dat tungg tho extremely cuuuuuute most angery pupper
+          I have ever seen wrinkler tungg most angery pupper I have ever seen, doggo heck wrinkler. Sub woofer the neighborhood
+          pupper wrinkler the neighborhood pupper, doge. Lotsa pats super chub the neighborhood pupper shooberino shoob, yapper tungg.
+          Boof pupper blep I am bekom fat pats fluffer, mlem sub woofer very good spot wrinkler. puggo sub woofer fat boi. Corgo
+          long water shoob fat boi long water shoob much ruin diet, lotsa pats doggorino.
+        </p>
+  </section>
+  <section class="blogthumb"><img src="images/blog-2.jpg" alt="blog photo" />
```
Comment (jimmynono) [inline blog.html]: Try to be more descriptive with your alt tags.

File: dogs.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        <li><a href="dogs.html">Dogs /</a></li>
+        <li><a href="blog.html">Blog</a></li>
+      </ul>
+    </div>
+
+<div class="here">
+  <h2>Here are all of our dogs!</h2>
+</div>
+
+<article class="dogprofiles">
```
Comment (jimmynono) [inline dogs.html]: I'd argue that the wrapping container is a section, and the dog tiles are more articles than sections (or a div)

File: styles/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
+z* {
```
Comment (jimmynono) [inline styles/stylesheet.css]: I think you have an extra z here.

File: styles/stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+/* NAVIGATION */
+
+nav {
+  background-color: #d1d1d1;
+  height: 3.9rem;
+}
+
+nav ul {
+  padding: 18px 0;
```
Comment (jimmynono) [inline styles/stylesheet.css]: just for consistency, keep to rems for padding/margins to stay with your font sizes. Just a little cleaner.

File: styles/stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+  justify-content: center;
+  align-content: space-between;
+  flex-direction: row;
+  flex-flow: wrap;
+  align-items: center;
+}
+
+.dog {
+  display: block;
+  width: 22em;
```
Comment (jimmynono) [inline styles/stylesheet.css:185]: These aren't exactly responsive. I'd lean you to using some sort of percent based system to keep them behaving properly. In your page I only see 2 dog tiles wide on a laptop, and we should see three. It also keeps them in ration with the rest of the page if you go with a percent based system.

File: styles/stylesheet.css [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+  padding: 3em;
+}
+
+.dogpic {
+  width: 100%;
+}
+
+/*stuff here */
```
Comment (jimmynono) [inline styles/stylesheet.css]: :)

File: styles/stylesheet.css (hunk 6 above)
Comment (kateblankenship1) [reply styles/stylesheet.css:185]: Okay! I shall work on integrating that when I work on the next assignment!

File: styles/stylesheet.css [hunk 8]
```diff
@@ -0,0 +1,446 @@
...
+  padding: 2em;
+}
+
+.footer-logo {
+  max-width: 20%;
+}
+
+/*BLOG PAGE */
+
+#blog-post-container {
```
Comment (jimmynono) [inline styles/stylesheet.css:254]: it's generally not good practice to use id's for CSS. It's totally approprate to create a single class. We try to use ID's for javascript needs.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+<!-- NAVIGATION -->
+  <nav>
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="dogs.html">Dogs</a></li>
+        <li><a href="blog.html">Blog</a></li>
+  </nav>
+<div class="format">
+  <div class="main-body">
+<div class="main-section">
```
Comment (jimmynono) [inline index.html:34]: Watch your indenting, it's a lot easier to see what's nested in elements if they are indented right.

**Pull request closed without merging**